server/tserver/src/main/java/org/apache/accumulo/tserver/FileManager.java

```diff
@@ -372,7 +368,8 @@ private void releaseReaders(KeyExtent tablet, List<FileSKVIterator> readers,
       for (FileSKVIterator reader : readers) {
         String fileName = reservedReaders.remove(reader);
         if (!sawIOException)
-          getFileList(fileName, openFiles).add(new OpenReader(fileName, reader));
```

Comment:
Possibly add a comment to document the method reason?

Reply:
For this one, the previous method was an undocumented private one-liner with an unhelpful name that was used only once. I inline'd it automatically with the IDE and verified that the automated inline was performed correctly.

I have no idea what any of this code in this entire class is trying to do, without possibly hours of investigation, as none of it has helpful comments at all. At this point, I don't think I could come up with anything other than the existing mediocre `put files in openFiles` comment at the top of the method. I'm disinclined to spend the time it would require to investigate enough to be capable of adding a more useful comment, so I would prefer to defer to the next meaningful refactor of this code. :smiley_cat: